modules/overview-of-distributed-workloads.adoc

```diff
@@ -36,6 +36,11 @@ Manages quotas and how distributed workloads consume them, and manages the queue
 
 You can run distributed workloads from data science pipelines, from Jupyter notebooks, or from Microsoft Visual Studio Code files.
 
+[NOTE]
+====
+Data Science Pipelines (DSP) workloads are not managed by the distributed workloads component, and are not included in the distributed workloads metrics.
```

Comment:
When referring to a specific component and also including the word "component", we should use initial caps:

"..the Distributed Workloads component,...."

Reply:
We do not use initcaps to refer to distributed workloads. We also should refer to it as the "distributed workloads feature" for consistency (my fault for providing the wrong text in the original ticket).

Reply:
Thanks for the advice, Breda. I have made a note of this for future reference.